### ISSUE TITLE
Search results for Ang page

### ISSUE THREAD

**Issue by navdeepsingh** [KhalisFoundation/sttm-web#1376]
**Describe the bug**
The search results do not give the latest results on selecting Type: Ang & selecting Source one by one.

**To Reproduce**
1. Go to https://www.sikhitothemax.org/ang?ang=12&source=D
2. Change to another source: GGSJ
3. Change to another source: Bhai Gurdas Ji Vaaran

**Expected behavior**
Same as for other types, the Ang page must filter correct results.

**Screenshots**
![Screen Shot 2021-04-05 at 5 03 17 PM](https://user-images.githubusercontent.com/2968787/113569948-fa7c9f80-9630-11eb-8929-e93dd4ed9930.png)

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Chrome
 - Version 89.0.4389.114

**Comment by tsingh777** [comment]
@navdeepsingh I believe one would have to click search for the filter to apply. I don't think we're updating page based on a filter change.

**Comment by navdeepsingh** [comment]
@tsingh777 You are right, Actually, page render when we change the selection, it gives the hint that results come in.
Also, this issue is just for Ang selection, either for other type selection without clicking at Search icon, it renders results.
